#python -m PyQt5.uic.pyuic -x GUI.ui -o Interface.py
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Scanfrm(object):
    def setupUi(self, Scanfrm):
        Scanfrm.setObjectName("Scanfrm")
        Scanfrm.setWindowModality(QtCore.Qt.ApplicationModal)
        Scanfrm.setEnabled(True)
        Scanfrm.resize(450, 475)
        font = QtGui.QFont()
        font.setFamily("Palatino Linotype")
        font.setBold(True)
        font.setWeight(75)
        Scanfrm.setFont(font)
        Scanfrm.setMouseTracking(False)
        Scanfrm.setAcceptDrops(False)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("Image/logo.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        Scanfrm.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(Scanfrm)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(140, 30, 941, 61))
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(18)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.Image = QtWidgets.QWidget(self.centralwidget)
        self.Image.setGeometry(QtCore.QRect(20, 110, 261, 281))
        self.Image.setObjectName("Image")
        self.groupBox = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox.setGeometry(QtCore.QRect(290, 110, 141, 281))
        self.groupBox.setTitle("")
        self.groupBox.setObjectName("groupBox")
        self.Openbtn = QtWidgets.QPushButton(self.groupBox)
        self.Openbtn.setGeometry(QtCore.QRect(20, 50, 101, 61))
        self.Openbtn.setAutoRepeat(True)
        self.Openbtn.setAutoExclusive(True)
        self.Openbtn.setAutoDefault(True)
        self.Openbtn.setDefault(False)
        self.Openbtn.setFlat(False)
        self.Openbtn.setObjectName("Openbtn")
        self.Savebtn = QtWidgets.QPushButton(self.groupBox)
        self.Savebtn.setGeometry(QtCore.QRect(20, 150, 101, 71))
        self.Savebtn.setAutoRepeat(True)
        self.Savebtn.setAutoExclusive(True)
        self.Savebtn.setAutoDefault(True)
        self.Savebtn.setDefault(False)
        self.Savebtn.setFlat(False)
        self.Savebtn.setObjectName("Savebtn")
        Scanfrm.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(Scanfrm)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 450, 26))
        self.menubar.setObjectName("menubar")
        self.menuMenu = QtWidgets.QMenu(self.menubar)
        self.menuMenu.setObjectName("menuMenu")
        self.menuEdit = QtWidgets.QMenu(self.menubar)
        self.menuEdit.setTitle("")
        self.menuEdit.setObjectName("menuEdit")
        Scanfrm.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(Scanfrm)
        self.statusbar.setObjectName("statusbar")
        Scanfrm.setStatusBar(self.statusbar)
        self.Openmn = QtWidgets.QAction(Scanfrm)
        self.Openmn.setObjectName("Openmn")
        self.Savemn = QtWidgets.QAction(Scanfrm)
        self.Savemn.setIconText("Save")
        self.Savemn.setObjectName("Savemn")
        self.Exitmn = QtWidgets.QAction(Scanfrm)
        self.Exitmn.setObjectName("Exitmn")
        self.Resizemn = QtWidgets.QAction(Scanfrm)
        self.Resizemn.setObjectName("Resizemn")
        self.Scanmn = QtWidgets.QAction(Scanfrm)
        self.Scanmn.setObjectName("Scanmn")
        self.menuMenu.addAction(self.Openmn)
        self.menuMenu.addAction(self.Exitmn)
        self.menubar.addAction(self.menuMenu.menuAction())
        self.menubar.addAction(self.menuEdit.menuAction())

        self.retranslateUi(Scanfrm)
        QtCore.QMetaObject.connectSlotsByName(Scanfrm)

    def retranslateUi(self, Scanfrm):
        _translate = QtCore.QCoreApplication.translate
        Scanfrm.setWindowTitle(_translate("Scanfrm", "SCANNING"))
        self.label.setText(_translate("Scanfrm", "X4FIT OCR"))
        self.Openbtn.setText(_translate("Scanfrm", "Open Image"))
        self.Savebtn.setText(_translate("Scanfrm", "Start"))
        self.menuMenu.setTitle(_translate("Scanfrm", "Menu"))
        self.Openmn.setText(_translate("Scanfrm", "Open Image"))
        self.Savemn.setText(_translate("Scanfrm", "Save"))
        self.Exitmn.setText(_translate("Scanfrm", "Exit"))
        self.Resizemn.setText(_translate("Scanfrm", "Resize"))
        self.Scanmn.setText(_translate("Scanfrm", "Scan"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Scanfrm = QtWidgets.QMainWindow()
    ui = Ui_Scanfrm()
    ui.setupUi(Scanfrm)
    Scanfrm.show()
    sys.exit(app.exec_())
